File: crackme.cpp
#include <iostream>
#include <string>
using namespace std;
int main() {

    string password = "crackme";
    string userInput;

    cout << "password: ";
    cin >> userInput;

    if (userInput == password) {
        cout << "success!" << endl;
    } else {
        cout << "fail." << endl;
    }

    return 0;
}

File: crackme.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace crackme
{
    class Program
    {
        static void Main(string[] args)
        {
            string password = "crackme";
            Console.WriteLine("password: ");
            string userInput = Console.ReadLine();
            if (userInput == password)
            {
                Console.WriteLine("success!");
            } else { Console.WriteLine("fail."); }
        }
    }
}